genshin/types.py

```diff
@@ -29,6 +29,9 @@ class Game(str, enum.Enum):
     HONKAI = "honkai3rd"
     """Honkai Impact 3rd"""
 
+    STARRAIL = "hkrpg"
```

Comment:
This string is based on what hoyolab uses internally. Though it probably doesn't really matter currently.

Reply:
You're right, I just didn't know what to use and figured that was a fine choice.